app/controllers/trains_controller.rb

```diff
@@ -1,75 +1,47 @@
 class TrainsController < ApplicationController
   before_action :set_train, only: [:show, :edit, :update, :destroy]
 
-  # GET /trains
-  # GET /trains.json
   def index
     @trains = Train.all
   end
 
-  # GET /trains/1
-  # GET /trains/1.json
   def show; end
 
-  # GET /trains/new
   def new
     @train = Train.new
   end
 
-  # GET /trains/1/edit
   def edit; end
 
-  # POST /trains
-  # POST /trains.json
   def create
     @train = Train.new(train_params)
-
-    respond_to do |format|
-      if @train.save
-        format.html { redirect_to @train, notice: 'Train was successfully created.' }
-        format.json { render :show, status: :created, location: @train }
```

Comment:
If you remove ability to return data in json, why you need view _show.json.jbuilder_ ?